app/ui-react/packages/api/src/useApiConnectorSummary.tsx

```diff
@@ -6,8 +6,7 @@ import { callFetch } from './callFetch';
 export function useApiConnectorSummary(
   specification: string,
   connectorTemplateId?: string,
-  portName?: string,
-  serviceName?: string
+  configured?: any
```

Comment:
What's the type for this?  Is it just a dictionary or something?  Be good to use something more helpful than any if possible.

Reply:
It's quite similar to the interface we were discussing [here](https://github.com/syndesisio/syndesis/pull/8656#discussion_r437563722), actually not sure why I didn't use that one depending on what we decide to do with it.